### PR TITLE
I didn't have the New / Scala class even after completing the first t…

### PR DESCRIPTION
**Pull request by ProductiveRage**
…utorial

When I followed the first tutorial, the IDEA project allowed me to add a new Scala class after I performed the "Add Framework Support" but I had to do it again when I created the sbt project a described in this step

### CODE REVIEW

**Comment by SethTisue** [comment]
is there an IntelliJ user who could review this?

**Comment by SethTisue** [comment]
pinging repo watchers — could an IntelliJ user review this, please?

File: _getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ but here's a glance at what everything is for:
 => `main`
 1. Right-click `scala` and select **New** => **Package**
 1. Name the package `example` and click **OK**.
-1. Right-click the package `example` and select **New** => **Scala class**.
+1. Right-click the package `example` and select **New** => **Scala class** (if you don't see this option, right-click the `SbtExampleProject`, click **Add Framewords Support**, select **Scala** and proceed)
```
Comment (lorenzpahl) [inline _getting-started/intellij-track/building-a-scala-project-with-intellij-and-sbt.md:58]: There is a spelling error:

```suggestion
1. Right-click the package `example` and select **New** => **Scala class** (if you don't see this option, right-click the `SbtExampleProject`, click **Add Framework Support**, select **Scala** and proceed)
```

**Comment by lorenzpahl** [comment]
I can confirm that this happens sometimes. The suggested way to resolve this with "Add Framework Support" worked for me.

**Comment by SethTisue** [comment]
@ProductiveRage can you respond to the review feedback and also rebase it? looks like this is about ready for merge

**Comment by ProductiveRage** [comment]
Sorry that there's been such a long delay on my replying.. and that there
is going to be a few more days yet, I'm afraid! But I *will *sort this out
as requested and get back to you soon.

**Comment by SethTisue** [comment]
still interested, now that the holidays have passed...?

**Comment by ProductiveRage** [comment]
I must be honest.. I'd been putting this off because I had only made the change to the documentation through the GitHub web UI and I don't believe that there's any way to rebase from within my PR. But I should be able to do this by cloning locally and rebasing using the command line there, so I'll sort that out soon.

Just in case I can continue being lazy, do maintainers not have the option "Rebase and merge" under the "Merge pull request" option? If not (or if there are any complications) then leave it with me and I'll do from my side!

**Comment by ProductiveRage** [comment]
@SethTisue Sorry about this, I think I might me making a meal out of something that should be fairly straight forward.. I just did performed the "resolve conflicts" action on the PR ahead of trying to rebase it and I think that it's now done a merge from the main into my fork and so I don't think that I can rebase it any more since this has intermingled the histories of the two branches?

On the one hand, I guess that this would make the PR merge easy but it would introduce the "Merge branch 'master' into patch-1" commit that I presume that you would rather avoid, which is why rebase was suggested?

I'll do my best to sort this out for the way in which you wish to maintain this project but I might need a little guidance.

**Comment by SethTisue** [comment]
This way is totally fine too. Thank you!

**Pull request merged**